packages/frontend/src/header/Header.scss

```diff
@@ -0,0 +1,29 @@
+@use '../global-variables' as gv;
+
+header {
+  /* Kinda messy, will order later */
+  background-color: gv.$bkg-color;
+  filter: drop-shadow(0 0px 5px #6b6b6b);
+  max-height: gv.$header-height;
+  min-width: 100vw;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  font-size: calc(10px + 2vmin);
```

Comment:
Is there a reason for this particular size? Generally font-size, if small-ish, is calculated as `em`s or `rem`s. Does there need to be a calc() here that's based on something below or above or can this be reduced to an `em`/`rem`?